NAMESPACE

```diff
@@ -3,4 +3,5 @@
 export(datasets)
 export(download)
 export(fetch)
+export(get_updates)
```

Comment:
Re the point below about exporting the `print` method for class `update_log`, this should show up here in an `S3method()` statement if you add `#' @export` immediately before you define the function.